.eslintrc.js

```diff
@@ -111,6 +111,14 @@ module.exports = {
         },
       ] },
     },
+    {
+      files: [
+        'lib/internal/modules/**/*.js',
+      ],
+      rules: {
+        'curly': 'error',
+      },
+    },
```

Comment:
Can you remove this to avoid getting error when backporting commits that landed before this rule was introduced?

Reply:
If there are commits to backport from before this rule, I highly recommend to backport them first.

Reply:
I don't know if they actually are any, regardless having less strict lint rules on LTS lines seems like a good idea to me, but it's just a nit

Reply:
So I looked into this, and the only two commits from before this rule are:

- https://github.com/nodejs/node/pull/47604, which is semver-major so we won’t backport it.
- https://github.com/nodejs/node/pull/49144, which @aduh95 requested not be backported. It also wouldn’t run afoul of this rule regardless, because it’s almost all deletions and the few additions include the newly required braces.

So I think the rule is fine to backport, and I’d rather not create unnecessary divergences from `main`. This rule is also very easy to abide by, and the linter can fix for it automatically.